### DISCUSSION TITLE
Changing the tabs

### DISCUSSION THREAD

**Discussion by ali14hasnain** [cotes2020/jekyll-theme-chirpy#1264] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

How do I change the names of the tabs instead of going to each index.html? I have tried using the site's index but it only changes the name of the tabs when I am on the home page. It reverts when I go to any other tab.

### Operations you have already tried

I have tried using the site's index but it only changes the name of the tabs when I am on the home page.

**Comment by kungfux** [comment, marked as answer]
To rename an existing tab, simply rename the file e.g. from `_tabs/archives.md` to `_tabs/timeline.md`.

**Reply by DmitriyFrogo** [reply]
> simply rename the file

Instead of renaming file, you can add title variable to it's [Front Matter](https://jekyllrb.com/docs/front-matter/)

**Reply by geraldtui** [reply]
Hi @DmitriyFrogo , adding the title does not update the Tab name on the sidebar only the page name. 

In my case, I'm trying to rename Categories to Docs

Renaming categories.md to docs.md under "_tabs" works, however, all the permalinks remain pointed to /categories/category-name

Is there a way to update the permalinks?

**Reply by DmitriyFrogo** [reply]
> Hi @DmitriyFrogo , adding the title does not update the Tab name on the sidebar only the page name.

@geraldtui Maybe you doing something wrong?

My example
I have ```_tabs\completed.md``` that leads to ```/completed/``` page and called not "completed", but name that I defined

![изображение](https://github.com/cotes2020/jekyll-theme-chirpy/assets/65223198/0750c1cc-e77d-4e61-9aaa-ea82fd48a598)

> Is there a way to update the permalinks?

"Categories" / "Posts" / "Tags" names defined in _config.yml
Also you can add "permalink" variable in front matter for separate posts

https://github.com/cotes2020/jekyll-theme-chirpy/blob/700fd5bad7272dd950f861e8550215cd8fafb413/_config.yml#L218

**Reply by DmitriyFrogo** [reply]
Also names predefined in locale file
I think, you have conflict between locale file and title
https://github.com/cotes2020/jekyll-theme-chirpy/blob/700fd5bad7272dd950f861e8550215cd8fafb413/_data/locales/en.yml#L14

**Reply by geraldtui** [reply]
Thanks @DmitriyFrogo , the below fixed my issue

1. Created `_data/locales/eng.yaml` with the following content. (All locale files are [here](https://github.com/cotes2020/jekyll-theme-chirpy/tree/master/_data/locales) for reference)
``` yaml
# The tabs of sidebar
tabs:
  # format: <filename_without_extension>: <value>
  home: Home
  categories: Docs
  tags: Tags
  archives: All Posts
  about: About
```

2. Renamed `_tabs/categories.md` to `_tabs/docs.md`. This step isn't necessary but its good to match the file name with the tab name
